source/configuration_manual/mail_crypt_plugin.rst

```diff
@@ -320,6 +320,8 @@ encrypt new mail, use ``mail_crypt_save_version=0``:
     mail_crypt_global_private_key = <server.key
   }
 
+.. _mail_crypt_acl_plugin:
```

Comment:
Should this link be used somewhere also?

Reply:
it is, line 14.

Reply:
Oh, is that how it works?